application/src/main/java/com/softserve/lv460/application/events/OnRegistrationCompleteEvent.java

```diff
@@ -0,0 +1,18 @@
+package com.softserve.lv460.application.events;
+
+import lombok.Getter;
+import com.softserve.lv460.application.entity.ApplicationUser;
+import org.springframework.context.ApplicationEvent;
+
+@Getter
+@SuppressWarnings("serial")
+public class OnRegistrationCompleteEvent extends ApplicationEvent {
+  private String appUrl;
+  private ApplicationUser user;
+
+  public OnRegistrationCompleteEvent(ApplicationUser user, String appUrl) {
```

Comment:
use @ AllArgsConstructor

Reply:
I can't because I should call super(user) in constructor. Is there any way to do it with lombok?